### DISCUSSION TITLE
Feature request: "Include Note" keyboard shortcut

### DISCUSSION THREAD

**Discussion by ehamminga** [TriliumNext/Trilium#1450] [Ideas] [answered]
What I gathered from [Issue 1410](https://github.com/zadam/trilium/issues/1410) "Include Note" does not have its own shortcut yet. It's a nice to have, but would this be difficult to implement?

**Comment by zadam** [comment, marked as answer]
hi, I checked it and it was not difficult in this specific case so I added the possibility to stable branch.

Since this is actually a feature request and not really a discussion I think better place in the future would be [issues](https://github.com/zadam/trilium/issues).
